app/components/burger-menu-nav.js

```diff
@@ -0,0 +1,19 @@
+import Component from '@ember/component';
+import { inject as service } from '@ember/service';
+
+export default Component.extend({
+
+  //props
+  navigations: service(),
+  itemsTop: null,
+  itemsBottom: null,
+
+  init() {
+    this._super(...arguments);
+    const nav = this.navigations.nav;
```

Comment:
Peut être récupérere directement
`const navBody = this.navigations.nav.data.body;` si le nav actuel n'est utilisé que pour ça

Reply:
Oui c'est vrai, merci !